cmd/api/main.go

```diff
@@ -58,7 +58,7 @@ func AddCommandTo(parent *cobra.Command) {
 	parent.AddCommand(cmd)
 }
 
-func run(cmd *cobra.Command, args []string) {
+func run(_ *cobra.Command, _ []string) {
```

Comment:
I don't like that this loses information on what the arguments are about. For `*cobra.Command`, it's reasonably clear. But `[]string` could be lots of things. Does the linter allow keeping the names and assigning to blank:

```
func run(cmd *cobra.Command, args []string) {
  _, _ = cmd, args
```

That's the common idiom for marking that a variable is not used, similar to void-casting in C/C++.
If the linter allows that, I would prefer that for all places where the purpose of the arguments is not immediately clear from the typing.